etc/run-supervisor.sh

```diff
@@ -35,7 +34,7 @@ radia_run redhat-docker
         export SIREPO_JOB_DRIVER_SBATCH_SIREPO_CMD=/global/homes/${3::1}/$3/.pyenv/versions/py3/bin/sirepo
         export SIREPO_JOB_DRIVER_SBATCH_SRDB_ROOT='/global/cscratch1/sd/{sbatch_user}/sirepo'
         export SIREPO_JOB_SUPERVISOR_SBATCH_POLL_SECS=15
-        export SIREPO_JOB_SUPERVISOR_URI=http://$2:8001
+        export SIREPO_JOB_DRIVER_SBATCH_SUPERVISOR_HOST=$2
```

Comment:
This still needs to be URI, not HOST, because we have to allow for https and proxy host (alpha/beta/prod) or http and localhost/port (dev or curl).